### DISCUSSION TITLE
PROBLEMA CON npm test JSIII.test.js

### DISCUSSION THREAD

**Discussion by marjosavi481** [atralice/Curso.Prep.Henry#126] [General]
saludos tengo un problema en mi archivo, al resorver el homework y aplico npm test JSIII.test.js no me salen los resultados si no solo me sale esto:

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        5.974 s
Ran all test suites matching /JSIII.test.js/i.
npm ERR! Test failed.  See above for more details.
PS C:\Users\ljmaster\Curso.Prep.Henry> npm test JSIII.test.js

**Comment by seteriano** [comment]
Hola!   Pudiste resolverlo? Justo he empezado hoy con los ejercicios y los tests no me funcionan.  El outcome es exactamente igual que el tuyo.   Muchas gracias.
